lib/util/util.ex

```diff
@@ -255,6 +255,29 @@ defmodule Util do
     Timex.format!(time, "{h12}:{m} {AM}")
   end
 
+  @doc """
+
+  Formats a number of minutes as a string. If the number of minutes is
+  greater than 60, then split it out into hours and minutes
+
+  ## Examples
+      iex> Util.format_minutes_duration(59)
+      "59 min"
+
+      iex> Util.format_minutes_duration(61)
+      "1 hr 1 min"
+  """
+  @spec format_minutes_duration(integer) :: String.t()
+  def format_minutes_duration(duration) do
+    case duration do
+      duration when duration >= 60 ->
+        "#{div(duration, 60)} hr #{rem(duration, 60)} min"
```

Comment:
question: If `rem(duration, 60)` happens to equal 0, would we want to omit the `0 min`?

Reply:
Woops! Thanks for catching that 😅

Reply:
[Fixed here, I think](https://github.com/mbta/dotcom/pull/2473/commits/67697deab55272b81fb109c8f6cf2b8fafe4698a)

Reply:
Actually, what DO we want here? I copy-pasta'ed the [original code from Ye Olde Trippe Planner](https://github.com/mbta/dotcom/blob/ebb97fc10975f2160c986e02c52ec32570048201/lib/dotcom_web/views/trip_plan_view.ex#L395-L404), so the previous version _did_ have the `0 min`. 

I asked in the standup thread.

Reply:
Confirmed that we want `1 hr`, not `1 hr 0 min`, so the PR as it is now should be ready!